invenio_records_resources/services/records/params/pagination.py

```diff
@@ -17,12 +20,22 @@ class PaginationParam(ParamInterpreter):
 
     def apply(self, identity, search, params):
         """Evaluate the query str on the search."""
+        options = self.config.search_pagination_options
+
+        default_size = options["default_results_per_page"]
+        default_max_results = options["default_max_results"]
+
+        params.setdefault("size", default_size)
```

Comment:
Comment: I like this much better than the previous approach - much cleaner in IMHO 👍

Reply:
Ah good! I know there are other ways to go about it, but I thought this one carried the clearest meaning.